spark/src/test/scala/org/apache/spark/sql/delta/rowid/RowIdCloneSuite.scala

```diff
@@ -70,6 +71,22 @@ class RowIdCloneSuite
     }
   }
 
+  test("self-clone an empty table") {
```

Comment:
nit: I think it's better to pull the config out of the test and have separate tests for each config value, as it makes it easier to figure out what broke.
Something like
```
for (enableRowIdsForSource <- BOOLEAN_DOMAIN) {
  test(s"self-clone an empty table, enableRowIdsForSource=$enableRowIdsForSource") {
```